src/service/AggregateTransactionService.ts

```diff
@@ -24,16 +24,24 @@ import { InnerTransaction } from '../model/transaction/InnerTransaction';
 import { MultisigAccountModificationTransaction } from '../model/transaction/MultisigAccountModificationTransaction';
 import { SignedTransaction } from '../model/transaction/SignedTransaction';
 import { TransactionType } from '../model/transaction/TransactionType';
+import { Address } from '../model/account/Address';
+import { RepositoryFactory } from '../infrastructure/RepositoryFactory';
+import { NetworkRepository } from '../infrastructure/NetworkRepository';
 
 /**
  * Aggregated Transaction service
  */
 export class AggregateTransactionService {
+    private readonly multisigRepository: MultisigRepository;
+    private readonly networkRepository: NetworkRepository;
     /**
      * Constructor
      * @param multisigRepository
      */
-    constructor(private readonly multisigRepository: MultisigRepository) {}
+    constructor(public readonly repositoryFactory: RepositoryFactory) {
```

Comment:
remove public readonly (it just a parm, not a a object field as the fields are the created repositories)